templates/letsencrypt/4/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: '2'
```

Comment:
Please, rename this file to templates/letsencrypt/4/docker-compose.yml.tpl in order go templating works.